### PR TITLE
Feature/webhooks

### PR DESCRIPTION
**Pull request by rockingrohit9639**

### CODE REVIEW

File: src/collections/appointment.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      dataType: 'reference',
+      path: 'users',
+      name: 'User',
+    }),
+    doctor: buildProperty({
+      dataType: 'reference',
+      path: 'doctors',
+      name: 'Doctor',
+    }),
+    start_time: buildProperty({
```
Comment (abinashpanda) [inline src/collections/appointment.ts]: Rename `start_time` -> `startTime` and same with `end_time`

File: src/collections/appointment.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        },
+      },
+    }),
+    prescriptionAction: buildProperty({
+      dataType: 'array',
+      name: 'Prescription Actions',
+      of: {
+        dataType: 'string',
+        enumValues: [
+          { id: 'BOOK_YOGA', label: 'Book Yoga' },
```
Comment (abinashpanda) [inline src/collections/appointment.ts]: Rename `BOOK_YOGA` to `BOOK_MEDITATION_SERVICE` as there would many more services in future.

File: src/types/appointment.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export type Appointment = {
+  user: any // @TODO: Figure out correct type
```
Comment (abinashpanda) [inline src/types/appointment.ts]: `user` can be `firebase.User.UserRecord` type. Right?

Comment (rockingrohit9639) [reply src/types/appointment.ts]: I found the correct type sir, it is `EntityReference` from `firecms`

**Pull request merged**